### DISCUSSION TITLE
feature: lora and model meta-data -> civitai compatible

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#436] [Q&A]
like this

‰PNG

   
IHDR     °   «Xo3  #tEXtparameters photo of a woman <lora:XYZ:1>
Negative prompt: sketch, drawing, cartoon, 
Steps: 60, Sampler: DPM++ 3M SDE Exponential, CFG scale: 4, Seed: 1496050147, Size: 672x1200, Model hash: dbeee57206, Model: ABC, Lora hashes: "XYZ: 4ebf5f0ab51f", Version: f0.0.17v1.8.0rc-latest-278-gbfee03d8×j¦   IDAT ...

@mcmonkey4eva
WHAT ??? writer  : D

i mean you close every discossion, every feature 
and don't tell us when the function will be ready?

**Comment by mcmonkey4eva** [comment]
What

**Comment by kalle07** [comment]
you know that every one must type here

closed  Discussions
![grafik](https://github.com/user-attachments/assets/3ccaddd8-008a-4bf8-829d-b1a0d4d8011e)

so that only VERY few people can participate in some useful ideas

OK what i mean is
that SWARM-UI PNG with internal meta-data lke 


‰PNG

   
IHDR  @  À   V°C   	pHYs  Ä  Ä•+  2tEXtparameters {
  "sui_image_params": {
    "prompt": "SteepSlopeTiddies. photo of a woman",
    "model": "Flux1DevFp8_v10_merge",
    "images": 12,
    "seed": 854800691,
    "steps": 25,
    "cfgscale": 1.0,
    "aspectratio": "2:3",
    "width": 832,
    "height": 1216,
    "sampler": "euler",
    "scheduler": "sgm_uniform",
    "loras": [
      "flux/nude/cumonfacelorav2",
      "flux/nude/SteepSlopeTiddies_breasts",
      "flux/nude/PantyJeans_Pull_down"
    ],
    "loraweights": [
      "0.8",
      "0.8",
      "1"
    ],
    "vae": "ae",
    "cliplmodel": "clip_l.safetensors",
    "txxlmodel": "t5xxl_enconly.safetensors",
    "negativeprompt": "",
    "swarm_version": "0.9.3.1"
  },
  "sui_extra_data": {
    "date": "2024-11-17",
    "generation_time": "390.24 (prep) and 55.86 (gen) seconds"
  }

is not civitai compliant

**Reply by mcmonkey4eva** [reply]
civitai supports swarmui metadata

**Reply by mcmonkey4eva** [reply]
And, again, most discussion related to SwarmUI is on discord, not here. You are continuing to spam in the wrong place

**Reply by kalle07** [reply]
for me discord is spam like the hell  : D
and it is not very comfortable and confusing
and why i need a secon accound if you are here 
and disussion and features and bugs - may you can close all here ^^
and the install information is here and the branches and all ^^
good helper is quit the game ...
